### PR TITLE
Feature/optin on signup

### LINKED ISSUE #127: Add opt in field to person
* Add to person end point
 * Add to ecrm export

### PR DESCRIPTION
**Pull request by brettminnie**
Closes #127

### CODE REVIEW

File: src/TransformCore/PHE/ExporterBundle/Model/Helper/RespondentHelper.php [hunk 1]
```diff
@@ -24,6 +24,7 @@ public static function extractRespondent(Person $person)
             "age"          => $person->getAge(),
             "gender"       => $person->getGender(),
             "emailAddress" => $person->getEmailAddress(),
+            "hasOptedIn"   => (true === $person->isSignedUp()) ? "true" : "false"
```
Comment (eddiejaoude) [inline src/TransformCore/PHE/ExporterBundle/Model/Helper/RespondentHelper.php]: boolean -> string?

Comment (brettminnie) [inline src/TransformCore/PHE/ExporterBundle/Model/Helper/RespondentHelper.php]: Asked this before, doesn't write correctly to csv if I don't do this

Comment (eddiejaoude) [inline src/TransformCore/PHE/ExporterBundle/Model/Helper/RespondentHelper.php]: I thought I had, but was just double checking

**Comment by brettminnie** [comment]
Really annoying, got around it with the other export by using the
serializer ;)

**Pull request merged**